packages/components/tests/dummy/app/templates/components/power-select.hbs

```diff
@@ -0,0 +1,107 @@
+{{page-title "PowerSelect Component"}}
```

Comment:
There's no link to this page in the "index", is it intentional? Otherwise probably should leave under "Forms" (something like `Form / PowerSelect` maybe?).

Reply:
Was not sure where to place it at the time